### ISSUE TITLE
Accessibility: Image picker not accessible when Voice over is turned on

### ISSUE THREAD

**Issue by Falcopus1986** [signalapp/Signal-iOS#4747]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Describe here the issue that you are experiencing.
I noticed, when I opened the photo gallery to select an image, all the images are inaccessible, I.E VoiceOver does not recognize the images as something I can interact with. Also, the close button is unlabeled.
### Steps to reproduce
- using hyphens as bullet points
- list the steps
- that reproduce the bug

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
Unable to select a specific picture to send.
**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)
You should be able to hear info about the pictures like the date and time the picture is made.
### Device info

**Device**: iPhone XS

**iOS version**: 14.3.0

**Signal version:** 5.2.5

**Comment by Igor3947** [comment]
I'm having exactly the same issue. I can't view the list of photos that I am trying to send.
https://user-images.githubusercontent.com/46590022/118427899-4002a280-b69c-11eb-8c26-58b9fbc0cc1f.MOV
Hope this clarifies the problem.
https://debuglogs.org/df3e9f6605ea58f26f8775532647e08ee480cfa0de3dcf4dc07a893dcef62d94.zip
